test/e2e/copy_from_image_test.go

```diff
@@ -126,31 +126,63 @@ func TestCopyImageInputToTarWithNonDistributableLayersFlagButContainsANonDistrib
 }
 
 func TestCopyAnImageFromATarToARepoThatDoesNotContainNonDistributableLayersButTheFlagWasIncluded(t *testing.T) {
-	env := BuildEnv(t)
-	imgpkg := Imgpkg{t, Logger{}, env.ImgpkgPath}
-	defer env.Cleanup()
+	t.Run("environment with internet", func(t *testing.T) {
```

Comment:
im not sure i understand the distinction of having two tests where one uses stopNetworkAccessToSourceRegistry and the other one does not. what are we trying to verify?